### PR TITLE
feat: add generateReleaseNotes and releaseNotes

### LINKED ISSUE #182: Adding new properties to createReleaseSchema to choose whether to generate release notes and inject custom release notes
We want to add two new properties to the `createReleaseSchema` in order to choose whether to generate release notes and inject custom release notes.

New properties to be added into `createReleaseSchema` and passed to `createRelease`:

- `generateReleaseNotes`: refers to GitHub property `generate_release_notes`
- `releaseNotes`: refers to GitHub property `body`

### PR DESCRIPTION
**Pull request by davideroffo**
Closes #182 

This PR adds two new properties to the createRelease schema in order to choose whether or not generate automatically the release notes and to inject custom release notes.

### CODE REVIEW

File: lib/routes.js [hunk 1]
```diff
@@ -17,7 +17,9 @@ const createReleaseSchema = {
     .additionalProperties(false) // important
     .prop('version', S.string().required())
     .prop('target', S.string())
-    .prop('name', S.string()),
+    .prop('name', S.string())
+    .prop('generateReleaseNotes', S.boolean())
```
Comment (simoneb) [inline lib/routes.js:21]: can't we use the default behavior here in the schema so that we don't have to put it in code?

File: index.test.js [hunk 2]
```diff
@@ -152,3 +152,53 @@ test('create release with no target specified', async t => {
...
+      {
+        version: 'v9.9.9',
+        generateReleaseNotes: true,
+        releaseNotes: 'my release notes',
+        owner: 'salmanm',
+        repo: 'smn-repo',
+      },
+      'some-token'
+    )
+  )
```
Comment (simoneb) [inline index.test.js:201]: use sinon.assert to get better error messages in case assertions fail

File: lib/routes.js (hunk 1 above)
Comment (davideroffo) [reply lib/routes.js:21]: what do you mean by "default behaviour"? Naming the props the same as GitHub expect them (e.g.: generate_release_notes)?

Comment (simoneb) [reply lib/routes.js:21]: default value :)

**Review by simoneb — APPROVED** [review]
LGTM 🚀

**Pull request merged**